packs/mlops/dagshub/config.yml

```diff
@@ -0,0 +1,45 @@
+id: d0e829a6-ede4-4933-9065-9f0c56f8aa7e
+# Name of the pack (required)
+name: mlops-dagshub
+
+# Displayed in the UI
+title: DagsHub
+
+# Long-form description of the pack (required)
+description: |
+  DagsHub is a platform for data scientists and machine learning engineers to version and sync their data, models, experiments, and code. It allows you and your team to easily share, review and reuse your work, providing a GitHub-like experience for machine learning. DagsHub is built on popular open-source tools and formats, making it easy to integrate with the tools you already use like New Relic.
+
+# Displayed in search results and recommendations. Summarizes a packs functionality.
+summary: |
+  Integrating DagsHub and New Relic enables you to analyze and monitor machine learning training metrics in real-time. You can visualize your metrics in a New Relic dashboard, create custom metrics and set alerts to notify you of events and incidents happening during your training runs.
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
+
+# Authors of the pack (required)
+authors:
+  - New Relic
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - mlops
+  - ml
+  - ai
+  - dagshub
+  - machine-learning
+  - machine learning
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - dagshub-mlops
```

Comment:
This installPlan will have to be created, similar to the example for Algorithmia here https://github.com/newrelic/newrelic-quickstarts/blob/main/install/mlops/algorithmia/install.yml.
Can you add that installPlan to the PR?